nservicebus/pipeline/unit-of-work.md

```diff
@@ -6,6 +6,43 @@ redirects:
 - nservicebus/unit-of-work-in-nservicebus
 ---
 
+## Using a transaction scope
+
+If your business transaction is spread across multiple handlers there is always a risk of partial updates since one handler might succeed in updating the data while other won't. To avoid this use a unit of work that wraps all handlers in a `TransactionScope` and makes sure that there are no partial updates. Use following code to enable a wrapping scope:
+
+snippet:UnitOfWorkWrapHandlersInATransactionScope
+
+NOTE: This requires the selected storage to support enlisting in transaction scopes.
+
+WARNING: This might escalate to a distributed transaction if data in different databases are updated.
+
+### Controlling transaction scope options
```

Comment:
`Required` is used here. We can certainly document it but would this knowledge be beneficial to the user?

This UoW always creates a new transaction scope and throws if an ongoing transaction is detected. In such setting `Suppress` does not seem to bring any benefit, and `RequiresNew` seems to be equivalent to `Requires`.

Does this make sense?